astropy/coordinates/transformations/graph.py

```diff
@@ -30,7 +23,7 @@
     FunctionTransformWithFiniteDifference,
 )
 
-__all__ = []
+__all__: list[str] = []
```

Comment:
Arrgghh, missed it: please, for modules like `coordinates` let's stick to what is done so far, which is either to define an `__all__` that captures what is used elsewhere in the module (and import `*` in `__init__.py`, or not to define it and import explicitly. For `representations`, we ended up with the latter, which is fine, but let's not add empty `__all__` until there is agreement on that being useful.

Reply:
SGTM.